solutions/standard/variables.tf

```diff
@@ -126,3 +126,43 @@ variable "skip_en_kms_auth_policy" {
   description = "Set to true to skip the creation of an IAM authorization policy that permits all Event Notification instances in the resource group to read the encryption key from the KMS instance."
   default     = false
 }
+
+########################################################################################################################
+# COS
+########################################################################################################################
+
+variable "destination_name" {
+  type        = string
+  description = "The Destintion name."
+  default     = null
```

Comment:
update var description as per previous comment, and give this a default value of "COS Destination"